sbin/pfdhcplistener

```diff
@@ -175,7 +177,21 @@ sub setup_global {
     return ;
 }
 
+our %NET_TYPES_FOR_EXTERNAL_QUEUE = (
+    'dhcp-listener' => 1,
+    'management' => 1,
+);
+
+=head2 which_send_queue
+
+Determine which send queue
 
+=cut
+
+sub which_send_queue {
+    my ($net_type) = @_;
+    return (any { exists $NET_TYPES_FOR_EXTERNAL_QUEUE{$_} } expand_csv($net_type) ) ? 'pfdhcplistener_external' : 'pfdhcplistener' ;
```

Comment:
From my testing %NET_TYPES_FOR_EXTERNAL_QUEUE is empty here which prevents the detection from working

Reply:
Fixed by 4dfa485